src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -1766,72 +1770,72 @@ private void SetParameterAttributes(MethodOrPropertySymbol methProp, ParameterIn
                     if (defType == typeof(byte))
```

Comment:
the if ladder could probably just be a switch on TypeCode. Not necessary in this change though.